mod/admin.php

```diff
@@ -1093,9 +1093,10 @@ function update_table(App $a, $table_name, $fields, $old_url, $new_url)
 		// update config
 		$configFileSaver = new \Friendica\Util\Config\ConfigFileSaver($a->getBasePath());
 		$configFileSaver->addConfigValue('config', 'hostname', parse_url($new_url, PHP_URL_HOST));
+		$configFileSaver->addConfigValue('system', 'urlpath', parse_url($new_url, PHP_URL_PATH));
+		$configFileSaver->addConfigValue('system', 'url', $new_url);
```

Comment:
Is this comma-safe?

Reply:
What do you mean? The `$new_url` is already used several times before for updates

Reply:
Ah, are you talking about the comma bug of the configsaver? No, not yet.

In the meantime, there are two bugs on my radar:
- no-comma at the end
- missing category will lead to skip adding